packages/admin/cms-admin/src/pages/pageSearch/usePageSearch.ts

```diff
@@ -29,16 +29,25 @@ export interface PageSearchApi {
 interface UsePageSearchOptions {
     tree: Map<string, GQLPageSearchFragment[]>;
     pagesToRender: PageTreePage[];
-    domain: string;
+    siteUrl: string;
     setExpandedIds: Dispatch<SetStateAction<string[]>>;
     onUpdateCurrentMatch: (pageId: string, pages: PageTreePage[]) => void;
 }
 
-const usePageSearch = ({ tree, domain, setExpandedIds, onUpdateCurrentMatch, pagesToRender }: UsePageSearchOptions): PageSearchApi => {
+const usePageSearch = ({ tree, siteUrl, setExpandedIds, onUpdateCurrentMatch, pagesToRender }: UsePageSearchOptions): PageSearchApi => {
     const [matches, setMatches] = useState<PageSearchMatch[] | null>(null);
     const [currentMatchIndex, setCurrentMatchIndex] = useState<number | null>(null);
     const [query, setQuery] = useState("");
 
+    let domainHost: string;
+
+    try {
+        domainHost = new URL(siteUrl).host;
+    } catch (error) {
+        console.error("Invalid siteUrl provided:", siteUrl, error);
+        domainHost = "";
```

Comment:
Maybe we should go with undefined instead of an empty string here, what do you think?